### DISCUSSION TITLE
Problems about Paper [ END-TO-END OPTIMIZED IMAGE COMPRESSION ]

### DISCUSSION THREAD

**Discussion by achel-x** [tensorflow/compression#94] [General]
Recently, I read the seriese of papers that how to end to end compress image. 
![image](https://user-images.githubusercontent.com/67953243/131514864-3df49d37-b7bb-4d8e-9eb3-4999a9d70a37.png)
In this picture, input image x transforms to latent representation y by analysis transform. My question is concerned with z, **how x transforms to z**, or what is the g_p transform. I found the description in paper named it as perceptual transform, but I can't understand what exactly it is. And **what is the relationship between z and y** ? 
I am not good at mathmatics, so it's tough for me to understand it from the formulas in your papers.

**Comment by jonaballe** [comment]
Hi,

in that paper z corresponds to a perceptual space, in which distances correlate better with perceived distances between images. For instance, this could be a colorspace such as YCbCr, or a more general perceptual space such as [NLP](https://arxiv.org/abs/1701.06641) or [PIM](https://arxiv.org/abs/2006.06752).

Hope this helps!
